mobile/test/common/integration/test_server_interface.cc

```diff
@@ -9,27 +9,36 @@ static std::weak_ptr<Envoy::TestServer> weak_test_server_;
 
 static std::shared_ptr<Envoy::TestServer> test_server() { return weak_test_server_.lock(); }
 
-void start_server(bool use_quic) {
+void start_server(Envoy::TestServerType test_server_type) {
   Envoy::ExtensionRegistry::registerFactories();
   strong_test_server_ = std::make_shared<Envoy::TestServer>();
   weak_test_server_ = strong_test_server_;
 
-  if (auto e = test_server()) {
-    e->startTestServer(use_quic);
+  if (auto server_interface = test_server()) {
```

Comment:
Let's please use the explicit type here (and elsewhere in this file) https://google.github.io/styleguide/cppguide.html#Type_deduction

Also since there is no longer and Interface class, I think i'd simply call this variable `serve` here (and elsewhere)

Reply:
This class is still called interface - I just renamed the objective-c class.  To be honest I could rename them all, but there's both a test_server class and a test_server_interface class so we'd have to come up with something else

Reply:
Which class is still called interface? It looks like the `test_server()` method returns `std::shared_ptr<Envoy::TestServer>`. So I'd just name the variable `server`.

Reply:
Think I misread your comment on friday, you're totally right and I'll change